hyperdome_client/hyperdome_client.py

```diff
@@ -279,70 +258,78 @@ def session(self):
         connection and retrieving new UID.
         """
-        self.server = self.servers[server]
         self.chat_window.clear()
         self.message_text_field.clear()
-        try:
-            if self.server.is_therapist:
-                self.session.post(f"{self.server.url}/therapist_signup",
-                                  data={"masterkey": "megumin",
-                                        "username": self.server.username,
-                                        "password": self.server.password})
-                self.session.post(f"{self.server.url}/therapist_signin",
-                                  data={"username": self.server.username,
-                                        "password": self.server.password})
-
-            else:
-                self.get_uid()
-                self.therapist = self.session.post(
-                    f"{self.server.url}/request_therapist",
-                    data={"guest_id": self.uid}).text
-                if self.therapist:
-                    self.is_connected = True
-        except Exception as e:
-            print(
-                ''.join(
-                    traceback.format_exception(
-                        type(e),
-                        e,
-                        e.__traceback__)))
-            Alert(
-                self.common,
-                "therapy machine broke",
-                QtWidgets.QMessageBox.Warning,
-                buttons=QtWidgets.QMessageBox.Ok)
+        if self.is_connected:
+            self.disconnect_chat()
+        if self.server_dropdown.currentIndex() == \
+           self.server_dropdown.count() - 1:
+            self.server_dropdown.setCurrentIndex(0)
+            self.start_chat_button.setEnabled(False)
+            self.server_add_dialog.exec_()
+        elif self.server_dropdown.currentIndex() != 0:
+            self.server = self.servers[self.server_dropdown.currentText()]
+            self.get_uid()
+
+    def start_chat(self):
+        @QtCore.pyqtSlot(str)
+        def after_start(counselor: str):
+            if not self.server.is_counselor and not counselor:
+                self.handle_error("No counselors available.")
+                self.start_chat_button.setEnabled(True)
+                return
+            if self.server.is_counselor:
+                self.uid = counselor
+            self.is_connected = True
+            self.get_messages_task = threads.GetMessagesTask(self.session,
+                                                             self.server,
+                                                             self.uid)
+            self.get_messages_task.setAutoDelete(False)
+            self.get_messages_task.signals.success.connect(
+                self.on_history_added)
+            self.get_messages_task.signals.error.connect(lambda: 0)
+            self.timer.start()
+            self.start_chat_button.setText("Disconnect")  # locale
```

Comment:
When I was writing against a linter for C++ our rules were also  double space before comment and I got nightmares for the number of times I failed builds because I had only one. So I have been trained to do it that way.

That being said I don't know what this means.

I think at one point I was going to flag every user facing string that was localized prior so I knew where to re-introduced language localization, but without following through at all it's pointless.

Fixed.